test/acceptance/tests/basic/terraform/basic-install/main.tf

```diff
@@ -159,20 +159,7 @@ module "test_client" {
       linuxParameters = {
         initProcessEnabled = true
       }
-      # Keep the client running for 10 seconds to validate graceful shutdown behavior.
-      entryPoint = ["/bin/sh", "-c", <<EOT
-/app/fake-service &
-export PID=$!
-trap "{ echo 'TEST LOG: on exit'; kill $PID; }" 0
-function onterm() {
-    echo "TEST LOG: Caught sigterm. Sleeping 10s..."
-    sleep 10
-    exit 0
-}
-trap onterm TERM
-wait $PID
-EOT
-      ]
+      command = ["/app/fake-service"]
```

Comment:
This part is not so seamless. Since this overrides the entrypoint, users will have to provide a `command` to specify how their container starts up - unless there's a default command in the image).

I haven't found a way at runtime to discover the default entrypoint from the image.

Reply:
Should document that requirement in the variable docs?